src/grpc_requests/aio.py

```diff
@@ -448,17 +455,21 @@ async def _get_service_names(self):
         services = tuple([s.name for s in resp.list_services_response.service])
         return services
 
-    async def get_file_descriptor_by_name(self, name):
+    async def get_file_descriptors_by_name(self, name):
```

Comment:
While I'll wager there's been limited engagement with the original method - I think for the sake of not breaking anyone who might be using it, we should keep the original method, but mark it deprecated with docs or a message pointing folks to use  your new method.

Same thing goes for the `get_file_descriptors_by_symbol`.

Reply:
That makes sense. Will do